rasa_nlu/server.py

```diff
@@ -73,14 +74,30 @@ def parse_get():
             except InvalidModelError as e:
                 return jsonify({"error": "{}".format(e)}), 404
 
+    @rasa_nlu_app.route("/version", methods=['GET'])
+    @requires_auth
+    def version():
+        return jsonify({'version': __version__})
+
+    @rasa_nlu_app.route("/serverlog", methods=['GET'])
+    @requires_auth
+    def serverlog():
+        with open(config['log_file'], 'r') as f:
+            return(f.read())
+
+    @rasa_nlu_app.route("/config", methods=['GET'])
+    @requires_auth
+    def rasaconfig():
+        return jsonify(rasa_nlu_config.as_dict())
```

Comment:
right I see, nevertheless this variable will only be visible when server is executed as main(e.g. if  started in a wsgi runner like gunicorn that main code wont be executed and the variable will not be available.). Since the config is passed in to the surrounding function, I think you can use `config.as_dict()`. what do you think?